fetch-in-react-practice/client/components/app.jsx

```diff
@@ -39,21 +39,32 @@ export default class App extends React.Component {
     * TIP: Use Array.prototype.concat to create a new array containing the contents
     * of the old array, plus the object returned by the server.
     */
-
+    const todos = this.state.todos;
```

Comment:
Careful, this is not a copy. Objects, arrays, functions, etc are pass-by-reference so this is just another name for the same location in memory. You can create a "shallow" copy of an array with `.slice()`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice